project-words.txt

```diff
@@ -0,0 +1,166 @@
+hasparent
```

Comment:
Please sort and add a linter check that it is sorted.

Is this list case sensitive?

Reply:
Made the fixes